### PR TITLE
QCElemental: init at 0.20.0

### PR DESCRIPTION
**Pull request by sheepforce**
###### Motivation for this change
Adds QCElemental python library. This is a scientific library, providing constants and tools for computational chemistry. Part of the packages @markuskowa and me want to merge from https://github.com/markuskowa/NixOS-QChem/issues/56.

(see also #124370)

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/qcengine/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{ buildPythonPackage, lib, fetchPypi, cacert, numpy, pydantic, pint,
+  networkx, pytestrunner, pytestcov, pytest
+} :
+
+buildPythonPackage rec {
+  pname = "qcelemental";
+  version = "0.19.0";
+
+  checkInputs = [ pytestrunner pytestcov pytest ];
+  propagatedBuildInputs = [ numpy pydantic pint networkx cacert ];
```
Comment (markuskowa) [inline pkgs/development/python-modules/qcengine/default.nix:10]: What is `cacert` need for here?

**Comment by markuskowa** [comment]
@GrahamcOfBorg build pythonPackages.qcelemental

File: pkgs/development/python-modules/qcengine/default.nix (hunk 1 above)
Comment (sheepforce) [reply pkgs/development/python-modules/qcengine/default.nix:10]: Older versions required it, as they downloaded molecular structures and constants from the web. The current versions apparently does not require it anymore. I've removed it.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 124375` at ab484473 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.qcelemental</li>
    <li>python39Packages.qcelemental</li>
  </ul>
</details>

**Pull request merged**